packages/remix/src/index.ts

```diff
@@ -0,0 +1,31 @@
+// This file should only export agnostic Javascript/Web Fetch API exports that
+// are available everywhere (server, client, etc.).  Anything relying on
+// platform-specific APIs should be exported from `index-platform.ts`
+
+export { Cookie, createCookie } from './lib/cookie'
+export {
+  createRouter,
+  createStorageKey,
+  del,
+  form,
+  get,
+  post,
+  put,
+  resources,
```

Comment:
We should have `resource` here as well, along with `head`, `options`, and `patch`. I'm thinking we should also have the longform names, `createGetRoute`, etc.